libs/translator/reader/utils.cpp

```diff
@@ -621,7 +621,17 @@ bool ReadNodeGraphAttr(const UsdPrim &prim, AtNode *node, const UsdAttribute &at
     VtValue value;
     if (attr && attr.Get(&value, time.frame)) {
         // RenderSettings have a string attribute, referencing a prim in the stage
-        std::string valStr = VtValueGetString(value, &attr);
+        std::string valStr;
+        // First check if the attribute is actually holding a "string" value
+        if (value.IsHolding<std::string>() || value.IsHolding<TfToken>() ||  value.IsHolding<SdfPath>())
+            valStr = VtValueGetString(value, &attr);
+        else if (attr.HasAuthoredConnections()) {
```

Comment:
should we test first for the connection ? and then the value ? as people tend to first put a value and then connection, and both are stored on the attribute, what do you think ?

Reply:
in theory these node graph attributes are just set as a string attributes. I could have not added the connection at all, I just added it as a fallback solution, in case we find an attribute of an unexpected type